packages/network-clients/src/clients/queryClient.ts

```diff
@@ -36,9 +42,9 @@ export class GraphqlQueryClient {
 
   // QUERY REGISTRY QUERY FUNCTIONS
 
-  async getIndexer(address: string): Promise<GetIndexer_indexer> {
+  async getIndexer(address: string): Promise<any> {
     const result = await wrapApolloResult(
-      this.explorerClient.query<GetIndexer>({
+      this.explorerClient.query<GetIndexerQuery, GetIndexerQueryVariables>({
```

Comment:
Should also support all options supported by useQuery, for exmaple: `pollInterval`

Reply:
This query function will only be used internally in the package. You will be able to import useQuery hook (from react-hooks) that supports pollInterval automatically.